tests/testutil/config.go

```diff
@@ -53,17 +53,29 @@ func (tc *TestConfig) Remove() {
 
 // GetContent returns the configs Data.Content as list of strings, newline separated
 func (tc *TestConfig) GetContent() []string {
-	var data []string
-	if val, ok := tc.ConfigMap.Data["content"]; ok {
-		if val != "" {
-			for _, s := range strings.Split(val, "\n") {
-				if s != "" {
-					data = append(data, s)
-				}
-			}
-		}
+	return tc.getContentData(tc.ConfigMap)
+}
+
+// KubectlGet retrieves the full test ConfigMap kubernetes object
+// CLI Command Run: kubectl get cm testname -n testing-ns -o json
+func (tc *TestConfig) KubectlGet() corev1.ConfigMap {
+	args := []string{"get", "cm", tc.ConfigMap.Name, "-n", testingNamespace, "-o", "json"}
+	resultString, err := KubectlCmd(args)
+	if err != nil {
+		tc.T.Fatalf("Failed to get ConfigMaps:  %v", err.Error())
 	}
-	return data
+	var results corev1.ConfigMap
+	err = json.Unmarshal([]byte(resultString), &results)
+	if err != nil {
+		tc.T.Fatalf("Failed to unmarshal ConfigMaps result: %s with error: %v", resultString, err.Error())
+	}
+
+	return results
+}
+
+// GetKubeContent returns the kubectl configmap's Data.Content as list of strings, newline separated
+func (tc *TestConfig) GetKubeContent() []string {
+	return tc.getContentData(tc.KubectlGet())
```

Comment:
Any reason we can't merge `KubectlGet` and `GetKubeContent` ?

Reply:
We can. I was thinking the GetKubeContent would be useful for the future in case any tests get added that want to validate something other than content. Would it make more sense to add it at that time?

Reply:
I would say yes, just to avoid premature optimization.